doc/todo/RELEASE

```diff
@@ -1,22 +1,9 @@
-This describes what to do for an release!
-
-= Info =
-
-Following shell expansions are used in this document:
- ~e           .. source of libelektra
- ~build       .. the build directory
- ~elektra     .. the directory above ~e and ~build
- ~elektra-ftp .. where ftp files go to
- ~elektra-doc .. where doc flies go to
-
+This describes what to do for a release:
 
 = TODO =
 
 - make sure no TODO is in this file (resolve them first!)
-- Simplify release!
-- Scripts for strace, version, run tests!
 - Check transitions in plugin status: especially from/to experimental
```

Comment:
This is to be merged with the text below: "Run scripts/dev/update-infos-status with arguments for heuristics (--auto)"